style.css

```diff
@@ -1,88 +1,98 @@
-*{
-    font-family: 'Poppins';
-    font-style: normal;
+* {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
 }
-ul{
-    list-style: none;
-    margin: 0;
-    padding: 0;
-    display: flex;
-    justify-content: space-between;
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: space-between;
 }
+
 .Content {
-    background-image: url(Asset/Headerbg@2x.png);
-    background-size:contain;
+  background-image: url(Asset/Headerbg@2x.png);
+  background-size: contain;
 }
+
 .connect {
-    color: #7F8CFF;
-    flex: none;
-    order: 0;
-    flex-grow: 0;
+  color: #7f8cff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
```

Comment:
- [ ] Great effort colleague for working so hard on your project, I would kindly suggest you stick to the Figma design as much as possible, This is a great requirement that we need to stick to. Currently your design doesn't look like what we have in the Figma design. Kindly work on it and be sure they look so perfect.

| Your Design 👇 | Figma Design 👇 |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/78634841/235738723-1992d5d0-01c2-4bbf-84a6-6d95c8ca286b.png) |  ![image](https://user-images.githubusercontent.com/78634841/235739100-a8414e34-ae20-4c59-b6fe-5ee645a41eaf.png) |

kindly remove the number 2 untop of the webpage too